### PR TITLE
Add actuator health endpoint to app starter

### PR DESCRIPTION
**Pull request by yinan-symphony**
The health endpoint uses the HealthService to provide the core components health status, as well as the DatafeedLoop running process status, if the running process counter is not updated in a fixed time range, the process is considered as 'DOWN'.

### Description
Closes #[ISSUE NUMBER]

Please put here the intent of your pull request.

### Dependencies
List the other pull requests that should be merged before/along this one.

### Checklist
- [ ] Referenced an issue in the PR title or description
- [ ] Filled properly the description and dependencies, if any
- [ ] Unit/Integration tests updated or added
- [ ] Javadoc added or updated
- [ ] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

File: docs/spring-boot/app-starter.md [hunk 1]
```diff
@@ -126,3 +127,23 @@ public class ExtAppSpringApplication {
...
+
+```
+management:
+  endpoint:
+    web:
+      exposure:
+        include: 'symphonyBdk'
+    health:
+      show-details: "ALWAYS"
+```
```
Comment (symphony-soufiane) [inline docs/spring-boot/app-starter.md:149]: we need maybe to update configuration.md too with this new added config: https://github.com/finos/symphony-bdk-java/blob/main/docs/configuration.md

File: symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/DatafeedLoop.java [hunk 2]
```diff
@@ -37,4 +37,11 @@ public interface DatafeedLoop {
      * @param listener a Datafeed event listener to be unsubscribed
      */
     void unsubscribe(RealTimeEventListener listener);
+
+  /**
+   * The timestamp of the last successful pulling
+   *
+   * @return timestamp value in long
+   */
+  long lastPullTimestamp();
```
Comment (symphony-soufiane) [inline symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/DatafeedLoop.java:46]: I don't think we want to expose this method in DF loop api to users.

**Review by symphony-soufiane — APPROVED** [review]
few comments.

File: symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/DatafeedLoop.java (hunk 2 above)
Comment (yinan-symphony) [reply symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/DatafeedLoop.java:46]: then how to get this timestamp in a generic way ? another way is to move the health check method inside the DatafeedLoop,  but i would say expose a last pull timestamp method in this interface is more relevant than exposing a health check method.

File: docs/spring-boot/app-starter.md (hunk 1 above)
Comment (yinan-symphony) [reply docs/spring-boot/app-starter.md:149]: Just check this configuration.md content, il looks like it s more like BDK configuration only, if we put this into configuration.md, i feel that it s kinda mix between BDK and spring boot config, especially considering that this config is in app-starter only.

**Pull request merged**